firebase.json

```diff
@@ -0,0 +1,16 @@
+{
+  "hosting": {
+    "public": "site",
+    "ignore": [
+      "firebase.json",
+      "**/.*",
+      "**/node_modules/**"
+    ],
+    "rewrites": [
```

Comment:
Can we change this rewrite to the below:
``` 
"rewrites": [
      {
        "regex": "^[\/\\w\\.-]+[^\\.]..$",
        "destination": "/index.html"
      }
    ]
```
Can also have this done in another PR